scheduling-manager-workflows/src/main/java/com/amazonaws/blox/schedulingmanager/deployment/steps/GetDeploymentData.java

```diff
@@ -34,19 +35,22 @@
   @Override
   public void handleRequest(InputStream input, OutputStream output, Context context)
       throws IOException {
-    log.debug("getDeploymentData lambda");
 
-    final DeploymentInput deploymentInput = encoder.decode(input, DeploymentInput.class);
+    final DeploymentWorkflowInput deploymentWorkflowInput =
+        encoder.decode(input, DeploymentWorkflowInput.class);
 
     log.debug(
         "deployment input name {} and account {}",
-        deploymentInput.getName(),
-        deploymentInput.getAccount());
+        deploymentWorkflowInput.getName(),
+        deploymentWorkflowInput.getAccount());
 
+    //TODO: retrieve from deployment table
     final DeploymentData deploymentData =
```

Comment:
What does the `Data` suffix mean? It doesn't seem to consistently mean "coming from a data store" (cf. `TaskData`), nor that it's representing some sort of raw data, so it's a bit confusing. Personally I'd omit it everywhere in favour of a more descriptive base name.

Reply:
suggestions?

Reply:
As per previous comment, we can revisit once we have more clarity on what inputs/outputs are needed here.